source/_posts/2023-09-06-release-20239.markdown

```diff
@@ -66,10 +52,10 @@ Don't forget to [join our release party live stream on YouTube](https://www.yout
 
 ## Climate, humidifier and water heater more info
 
-The more info for climate, humidifier and water heater entities got an update, if you ask me, they look absolutely stunning!
-They now have a newly designed circular slider, that shows the current temperature and the setpoint(s).
-There are also buttons to change the setpoint.
-It will glow in the middle to show it's current action, as well as show that.
+The more info screen for climate, humidifier and water heater entities got an update, if you ask me, it looks absolutely stunning!
+They now have a newly designed circular slider, that shows the current temperature and the target temperature. And allows you to swipe to the desired target temperature easily.
```

Comment:
```suggestion
It now has a newly designed circular slider, that shows the current temperature and the target temperature. And allows you to swipe to the desired target temperature easily.
```

Reply:
It being "_The more info screen for climate, humidifier and water heater entities_"